scripts/run-gtest.ps1

```diff
@@ -708,7 +718,11 @@ try {
     Log "$($TestCount) test(s) run."
     if ($KeepOutputOnSuccess -or ($TestsFailed -ne 0) -or $AnyProcessCrashes) {
         Log "Output can be found in $($LogDir)"
-        Write-Error "$($TestsFailed) test(s) failed."
+        if ($ErrorsAsWarnings) {
+            Write-Host "$($TestsFailed) test(s) failed."
```

Comment:
Theres actually a Write-Warning you should be able to use here.

Reply:
Done.